Network Class/Network.cpp

```diff
@@ -138,6 +138,39 @@ void Network::updateWeightsAndBiases()
 
 }
 
+//moves backwards through Network calculating error at each level and using that to increment sumNablaB and sumNablaW
+//requires the assumption that expected values has an int
+bool backProp(int index)
+{
+	int lastInd = numLayers - 1;
+	int lastSize = layerSizes[lastInd];
+	Vector expectedValues = getAt<int>(expectedValuesInfile, index);
+	
+	bool correct = true;
+	// to write: check if activations[lastInd] matches expectedVal at index and set correct accordingly
+
+	// Matrix costPrime = activations[lastInd] - expectedValues;
+	// above statement needs expectedValues to be a matrix to work
+	// consider the implications of having getAt return a (j x 1) matrix instead of a vector
+	Matrix costPrime; costPrime.set_size(lastSize, 1);
+	for (int i = 0; i < lastSize; i++
+		 costPrime(i,0) = activations[lastInd](i,0) - expectedValues[i];
```

Comment:
costPrime() has to be a function, outside the class. Cost Function is something the programmers in future, or even we, will be changing later!

Reply:
i've added the modified code for using a cost prime func in a comment in the newest commit and will be changing it when we have a cost prime func written.

Reply:
Alright! And also, the backprop will always return a true. Make sure to change that too

Reply:
Alright! And also, the backprop will always return a true. Make sure to change that too